src/List/ListItem.js

```diff
@@ -676,16 +687,16 @@ class ListItem extends Component {
           simpleLabel ? this.createLabelElement(styles, contentChildren, other) :
           disabled ? this.createDisabledElement(styles, contentChildren, other) : (
             <EnhancedButton
-              containerElement="span"
               {...other}
+              containerElement={containerElement}
```

Comment:
It's overridden by simply giving a `containerElement` prop. It's pulled out from the `other` props because it was being passed on to the label or disabled element if it wasn't pulled out. There's no logic where anything in `other` would be overriding `containerElement`.